Header Header
float64 Latitude
float64 Longitude	
float64 Altitude
float64 UTM_northing
float64 UTM_easting
int64 Zone
string Letter
string GPGGA
sensor_msgs/Imu IMU
sensor_msgs/MagneticField MagField
string VNYMR

